apps/consent/env.ts

```diff
@@ -8,10 +8,16 @@ export const env = createEnv({
   },
   shared: {
     GRAPHQL_ENDPOINT: z.string().default("http://localhost:4002/graphql"),
+    OTEL_EXPORTER_OTLP_ENDPOINT: z.string().default("http://localhost:4318"),
+    TRACING_SERVICE_NAME: z.string().default("consent"),
+    NEXT_RUNTIME: z.string(),
   },
   runtimeEnv: {
     CORE_AUTH_URL: process.env.CORE_AUTH_URL,
     HYDRA_ADMIN_URL: process.env.HYDRA_ADMIN_URL,
     GRAPHQL_ENDPOINT: process.env.GRAPHQL_ENDPOINT,
+    OTEL_EXPORTER_OTLP_ENDPOINT: process.env.OTEL_EXPORTER_OTLP_ENDPOINT,
+    TRACING_SERVICE_NAME: process.env.TRACING_SERVICE_NAME,
+    NEXT_RUNTIME: process.env.NEXT_RUNTIME,
```

Comment:
```suggestion
    NEXT_RUNTIME: process.env.NEXT_RUNTIME,
```

I would have an exception with NEXT env variable. I would not add them here. 

also, sometime the otel code, because it's loaded before the rest of the app, would not have access to this env.ts or other part of the code. but I don't really understand the nuances of this on my end